.gitignore

```diff
@@ -3,3 +3,4 @@
 /build
 /.phpcs-cache
 /.phpunit.result.cache
+/.idea
```

Comment:
This is unrelated to this repository.

You should ignore the .idea directory on a global level using the git config option core.excludesfile. See http://egorsmirnov.me/2015/05/04/global-gitignore-file.html